store/store.go

```diff
@@ -305,51 +304,76 @@ func (s *Store) hasByHeight(height uint64) (bool, error) {
 	return exists(pathODS)
 }
 
-func (s *Store) Remove(ctx context.Context, height uint64, datahash share.DataHash) error {
+func (s *Store) RemoveAll(ctx context.Context, height uint64, datahash share.DataHash) error {
```

Comment:
Let's name to `RemoveODSQ4`. Its not clear what All stands for. 

Same for helpers methods and metrics.